js/components/slider.js

```diff
@@ -0,0 +1,87 @@
+Fliplet.FormBuilder.field('slider', {
+  name: 'Slider',
+  category: 'Advanced',
+  props: {
+    placeholder: {
```

Comment:
There's no `placeholder` for the slider field. Let's remove this. Looks like there's nothing referencing it anyway.

```suggestion
```